.env.sample

```diff
@@ -2,7 +2,7 @@
 NODE_HOSTNAME=
 
 # Deployment env vars
-NETWORK_NAME=testnet-validated
+NETWORK_NAME=testnet-beta
```

Comment:
This will make the network default to Beta TestNet. We would need to update the docs. Should we keep alpha testnet as default instead?

Reply:
Beta should be the default